lib/plugins/aws/deploy/compile/events/apiGateway/lib/validate.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
           let path;
 
           if (typeof event.http === 'object') {
-            method = event.http.method;
+            method = event.http.method.toLowerCase();
```

Comment:
This will only consider the case that the event type is an object. Isn't it also required if the event type is a simple string?